app/api/chat/router.py

```diff
@@ -0,0 +1,40 @@
+from uuid import UUID
+from starlette import status
+from fastapi import APIRouter, Depends, HTTPException
+from app.api.chat.schemas import ChatResponse
+from app.api.user.service import get_current_active_user
+from app.core.utils import db_dependency
+from app.models import Chat
+
+router = APIRouter()
+
+@router.post('/{id}', status_code=status.HTTP_201_CREATED, response_model=ChatResponse, dependencies=[Depends(get_current_active_user)])
```

Comment:
```suggestion
@router.post('/{contact_id}', status_code=status.HTTP_201_CREATED, response_model=ChatResponse, dependencies=[Depends(get_current_active_user)])
```
é bom deixar bem claro o que a variável é